var/spack/repos/builtin/packages/py-neo4j/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
```

Comment:
Today the license header changed. See https://github.com/spack/spack/pull/48352.